src/applications/personalization/profile/sass/notification-radio-buttons.scss

```diff
@@ -0,0 +1,69 @@
+.rb-fieldset-input {

```

Comment:
a few comments:

- there's a lot of repetition in this file that could be cleaned up
- but, more importantly, we should also move a lot of the styling out of this scss file and into the markup itself using our `vads-u` utility classes, which is our normal best practice

But, since this is an experiment and a WIP, I'm okay leaving this as is for now. I suspect we'll be doing more work in here over the next couple of weeks 🤞 and we can clean things up and refine then.

Reply:
sorry I need to get more familiar with those classes. I don't think I realized that was a standard practice. I also wrote those just in case they may want to use this in the components repo, however I removed things like the ones with child elements because we had no use for them. If they wanted to use this every where else I would believe we would need to do a lot of work regardless.